tests/Authentication/ClientCredentialsEnvironmentAuthenticatorTest.php

```diff
@@ -117,6 +117,9 @@ public function testAuthenticate()
         /** @var GuzzleClientFactory $factory */
         $auth = new ClientCredentialsEnvironmentAuthenticator($factory, 'https://vault.azure.net');
         $token = $auth->getAuthenticationToken();
```

Comment:
nemel by i tady byt self::assertCount(1, $requestHistory);? call me hnidopich, ale takle nevis, jestli se ten http request udelal prvnim nebo druhym zavolanim getAuthenticationToken, takze bych to cekal za obema, jako ze po prvnim zavolani je 1 http request a po druhym taky (zadny request nepribyl)

Reply:
No tak přišlo mě, že když je tam po dvou zavoláních jeden, tak je to snad jasný :) Ale dobře no